src/Carousel.vue

```diff
@@ -1,13 +1,13 @@
 <template>
   <section class="VueCarousel">
-    <div class="VueCarousel-wrapper" ref="VueCarousel-wrapper">
-      <div
-        ref="VueCarousel-inner"
+    <div class="VueCarousel-wrapper"
+      ref="VueCarousel-wrapper">
+      <div ref="VueCarousel-inner"
         class="VueCarousel-inner"
         role="listbox"
         :style="{
           'transform': `translate3d(${currentOffset}px, 0, 0)`,
-          'transition': !dragging ? transitionStyle : 'none',
+          'transition': dragging ? 'none' : transitionStyle,
```

Comment:
Good fix here this is much nicer

Reply:
Oh, sorry, I formatted the code style with sublime and now it causes so much differences.